redis/src/aio/multiplexed_connection.rs

```diff
@@ -520,15 +580,180 @@ impl MultiplexedConnection {
         self.response_timeout = Some(timeout);
     }
 
+    #[cfg(feature = "cache-aio")]
+    async fn handle_cached_mget<'a>(
+        &mut self,
+        cmd: &Cmd,
+        ci: &CommandCacheInformationByRef<'a>,
+    ) -> RedisResult<Value> {
+        let mut response: Vec<Value> = vec![];
```

Comment:
please document what this function does. Its huge and unclear.

Reply:
I've documented and refactored it, I believe it's better now, can you check it again please ?